path.py

```diff
@@ -0,0 +1,28 @@
+import osmnx as ox
```

Comment:
ok will do that

Reply:
But leave the file for now Don't delete we might require for further reference in case let's push it to a folder name archived or some relevant name

Reply:
@healertrix  then put it in archive folder